### PR TITLE
Section 3: Storage Accounts

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: policy.hcl [hunk 1]
```diff
@@ -16,9 +16,101 @@ policy "cis-v1.30" {
...
+        WHERE delete_retention_policy_enabled IS NULL OR NOT delete_retention_policy_enabled
+      EOF
+    }
+
+    query "3.9" {
+      description = "Azure CIS 3.9: Ensure storage for critical data are encrypted with Customer Managed Key"
+      query = file("queries/manual.sql")
+    }
+
+    query "3.10" {
```
Comment (roneli) [inline policy.hcl:105]: are you sure this is manual?  we can checking blob service logging

File: policy.hcl [hunk 2]
```diff
@@ -16,9 +16,101 @@ policy "cis-v1.30" {
...
+    query "3.8" {
+      description = "Azure CIS 3.8: Ensure soft delete is enabled for Azure Storage"
+      query =<<EOF
+        SELECT
+        FROM azure_storage_accounts a LEFT OUTER JOIN azure_storage_blob_services b ON a.cq_id=b.account_cq_id
+        WHERE delete_retention_policy_enabled IS NULL OR NOT delete_retention_policy_enabled
+      EOF
+    }
+
+    query "3.9" {
```
Comment (roneli) [inline policy.hcl:100]: Can this be automated?

**Review by roneli — CHANGES_REQUESTED** [review]
looks good, some changes required

File: policy.hcl (hunk 1 above)
Comment (irmatov) [reply policy.hcl:433]: Implemented this query

File: policy.hcl (hunk 2 above)
Comment (irmatov) [reply policy.hcl:428]: Done

**Review by roneli — APPROVED** [review]

**Pull request merged**